packages/events/src/router/event/event.actions.declare.test.ts

```diff
@@ -26,3 +27,68 @@ test('Validation error message contains all the offending fields', async () => {
 
   await expect(client.event.actions.declare(data)).rejects.matchSnapshot()
 })
+
+test('Allows passing fields that are conditionally removed', async () => {
```

Comment:
Same here – Title is about allowing something but expect statement expects a reject